mqr/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class MqrStrata(models.Model):
+    province = models.CharField(max_length=25, null=False, blank=False)
+    weeks_pregnant = models.CharField(max_length=12, null=False, blank=False)
+    age = models.IntegerField(null=False, blank=False)
```

Comment:
Make this a charfield too since it needs to be a bucket/range, see other comment for more details